src/lib389/lib389/replica.py

```diff
@@ -1650,6 +1650,9 @@ def get_ruv(self):
                 escapehatch='i am sure')[0]
             data = ensure_list_str(ent.getValues('nsds50ruv'))
         except IndexError:
+            # There are no ruv elements, it's okay
+            pass
+        except ldap.NO_SUCH_OBJECT:
             # There is no ruv entry, it's okay
             pass
```

Comment:
The BZ (https://bugzilla.redhat.com/show_bug.cgi?id=2208104#c8) is related to a invalid replication setup. A replica containing a RUV entry but no replicated suffix entry. So the RUV can not be found with a get_ruv search (err=32).

Can this patch be used for such invalid setup ?

Reply:
I think that trapping the exception would prevent the exception but people will still be puzzled to see an empty ruv.
I would have expected that a message was also added in get_ruv (li_conf/replication.py) if there are no ruvs to warn the admin that the db is likely uninitialized.

Reply:
But I am reporting that the replica needs to be initialized.  Am I missing something?